azure-function/src/main.tf

```diff
@@ -1,7 +1,48 @@
+# helpful
+# https://faun.pub/managed-identities-in-azure-with-terraform-28ca062ab786
+
+locals {
+  policies = {
+    "read" = {
+      role  = "Microsoft.Authorization/read"
+      scope = ""
+    }
+  }
+}
+
+// start mdxc
```

Comment:
Why are we adding these resources?

Reply:
This was just an example of using a user managed identity, assigning roles to that and using that as the app identity.